tests/subsys/bluetooth/fast_pair/crypto/testcase.yaml

```diff
@@ -9,20 +9,20 @@ tests:
     extra_args: CONF_FILE=prj_mbedtls.conf
   fast_pair.crypto.oberon:
     platform_allow:
-      nrf52dk_nrf52832 nrf52840dk_nrf52840 nrf5340dk_nrf5340_cpuapp nrf9160dk_nrf9160_ns
+      nrf52dk_nrf52832 nrf52840dk_nrf52840 nrf5340dk_nrf5340_cpuapp nrf5340dk_nrf5340_cpuapp_ns
```

Comment:
Could we list the items in separate lines as we do in the remaining configurations?